2b_portfolio_prep_supervisor_workflow.R

```diff
@@ -12,7 +12,34 @@ prep_pf <- raw_pf %>%
 
 prep_pf <- add_meta_portfolio(prep_pf, inc_meta_portfolio)
 
-port_raw_all_eq <- prep_pf %>%
+# taken from add_portfolio_flags()
+prep_pf <- check_isin_format(prep_pf)
+prep_pf <- check_valid_value_usd(prep_pf)
+
+# taken from add_flags()
+prep_pf <- prep_pf %>%
+  mutate(
+    flag = case_when(
+      !has_valid_value_usd ~ "Negative or missing input value",
+      !has_valid_isin ~ "Invalid or missing ISIN",
+      TRUE ~ "Included in analysis"
+    )
+  )
+
+# taken from overall_validity_flag()
+portfolio_total <- prep_pf %>%
+  mutate(valid_input = case_when(
+    !has_valid_value_usd ~ FALSE,
+    !has_valid_isin ~ FALSE,
+    TRUE ~ TRUE
+  ))
+
+portfolio_overview <- portfolio_summary(portfolio_total)
+
+portfolio_overview %>% write_rds(file.path(proc_input_path, paste0(project_name, "_overview_portfolio.rda")))
```

Comment:
if possible, please save RDS files with an ".rds" extension

Reply:
I would much prefer changing all of these at once for the entire PACTA repo, rather than just doing it here. This file will be saved as an rda in other workflows as well after all. Maybe it's time for a clean up soon?

Reply:
yeah, I'm all too aware that this is unavoidable in some cases... just always have to say it when I see it 😬